controller/network/path.go

```diff
@@ -50,6 +50,7 @@ func (network *Network) shortestPath(srcR *Router, dstR *Router) ([]*Router, int
 			if _, found := unvisited[r]; found {
 				cost := int64(r.CostFactor)
 				if l, found := network.linkController.leastExpensiveLink(r, u); found {
```

Comment:
We were including router cost factor (which can't actually be set anywhere) instead of link cost. Do we want to leave router cost factor here, and flesh it out later, or drop it?

Reply:
I'm not sure what you mean by "router cost factor". That's not something that existed before `Xt`. Link cost has always been included and should continue to be included, as it's currently the main externally-configured cost input.

Reply:
Router has a CostFactor which has existed for as long back as I could trace it in git. It's not set by anything other than the constructor. I think we should probably drop it unless you think the implementation will be fleshed out soon.

Reply:
Agreed. Drop it.